feature/mpls/sr/otg_tests/srte_egress_node_forwarding/egress_node_forwading_test.go

```diff
@@ -0,0 +1,674 @@
+	mplsHdrV4.BottomOfStack().SetValue(mplsSBit)
+	mplsHdrV4.TimeToLive().SetValue(mplsOuterTTL)
+
+	ipHdrV4 := flowV4.Packet().Add().Ipv4()
+	ipHdrV4.Src().SetValue(ateTrafficSrcIPv4)
+	ipHdrV4.Dst().SetValue(ateTrafficDstIPv4)
+
+	tcpHdrV4 := flowV4.Packet().Add().Tcp()
+	tcpHdrV4.SrcPort().SetValue(10000 + (packetsPerFlow % 50000))
+	tcpHdrV4.DstPort().SetValue(tcpDstPort)
+
+	udpHdrV4 := flowV4.Packet().Add().Udp()
+	udpHdrV4.SrcPort().SetValue(10000 + (packetsPerFlow % 50000))
+	udpHdrV4.DstPort().SetValue(udpDstPort)
+
+	// Traffic Flow 2: Ethernet+MPLS+IPv6+Payload
+
+	t.Logf("Configuring ATE Flow: %s", flowV6Name)
+	flowV6 := otgConfig.Flows().Add().SetName(flowV6Name)
+	flowV6.Metrics().SetEnable(true)
+	flowV6.TxRx().Device().SetTxNames([]string{atePort1.Name + ".IPv6"}).SetRxNames([]string{atePort2.Name + ".IPv6"})
+	flowV6.Size().SetFixed(trafficFrameSize)
+	flowV6.Rate().SetPps(trafficRatePps)
+	flowV6.Duration().FixedPackets().SetPackets(packetsPerFlow)
+
+	ethHdrV6 := flowV6.Packet().Add().Ethernet()
+	ethHdrV6.Src().SetValue(atePort1.MAC)
+	ethHdrV6.Dst().Auto()
+
+	mplsHdrV6 := flowV6.Packet().Add().Mpls()
+	mplsHdrV6.Label().SetValue(mplsLabelV6)
+	mplsHdrV6.TrafficClass().SetValue(mplsExpBit)
+	mplsHdrV6.BottomOfStack().SetValue(mplsSBit)
+	mplsHdrV6.TimeToLive().SetValue(mplsOuterTTL)
+
+	ipHdrV6 := flowV6.Packet().Add().Ipv6()
+	ipHdrV6.Src().SetValue(ateTrafficSrcIPv6)
+	ipHdrV6.Dst().SetValue(ateTrafficDstIPv6)
+
+	tcpHdrV6 := flowV4.Packet().Add().Tcp()
+	tcpHdrV6.SrcPort().SetValue(10000 + (packetsPerFlow % 50000)) // pseudo-random src port > 1024
+	tcpHdrV6.DstPort().SetValue(tcpDstPort)
+
+	udpHdrV6 := flowV6.Packet().Add().Udp()
+	udpHdrV6.SrcPort().SetValue(20000 + (packetsPerFlow % 40000)) // pseudo-random src port > 1024
+	udpHdrV6.DstPort().SetValue(udpDstPort)
+
+	return otgConfig
+}
+
+func enableCapture(t *testing.T, config gosnappi.Config, port string) {
+	config.Captures().Clear()
+	t.Log("Enabling capture on ", port)
+	config.Captures().Add().SetName(port).SetPortNames([]string{port}).SetFormat(gosnappi.CaptureFormat.PCAP)
+
+}
+
+func startCapture(t *testing.T, otg *otg.OTG) gosnappi.ControlState {
+	t.Helper()
+	cs := gosnappi.NewControlState()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.START)
+	otg.SetControlState(t, cs)
+
+	return cs
+}
+
+func stopCapture(t *testing.T, otg *otg.OTG, cs gosnappi.ControlState) {
+	t.Helper()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.STOP)
+	otg.SetControlState(t, cs)
+}
+
+func processCapture(t *testing.T, otg *otg.OTG, port string) string {
+	bytes := otg.GetCapture(t, gosnappi.NewCaptureRequest().SetPortName(port))
+	time.Sleep(30 * time.Second)
+	capture, err := os.CreateTemp("", "pcap")
+	if err != nil {
+		t.Errorf("ERROR: Could not create temporary pcap file: %v\n", err)
+	}
+	if _, err := capture.Write(bytes); err != nil {
+		t.Errorf("ERROR: Could not write bytes to pcap file: %v\n", err)
```

Comment:
Since this is required for test setup, these should be `t.Fatalf`
```suggestion
		t.Fatalf("ERROR: Could not create temporary pcap file: %v\n", err)
	}
	if _, err := capture.Write(bytes); err != nil {
		t.Fatalf("ERROR: Could not write bytes to pcap file: %v\n", err)
```